src/controllers/organizations.js

```diff
@@ -283,10 +283,34 @@ async function updateOrgProfile(req, res) {
     }
 }
 
+async function unFollowOrga(req, res) {
```

Comment:
unfollowOrganization is a better naming i think since unfollow is one word and org can mean different things